package.json

```diff
@@ -1,25 +1,18 @@
 {
   "name": "makara-browserify",
-  "version": "1.2.3",
+  "version": "1.2.5",
```

Comment:
This line just brings the version up to date with the current version. When the change is merged, I will use `npm version patch` to increment the version. I will also add the final patch release version to the changelog.